core/src/main/java/io/opentelemetry/android/OpenTelemetryRumBuilder.java

```diff
@@ -552,14 +594,23 @@ private SpanExporter buildSpanExporter() {
         return spanExporterCustomizer.apply(defaultExporter);
     }
 
+    private MetricExporter buildMetricExporter() {
+        // TODO: Default to otlp...but how can we make endpoint and auth mandatory?
```

Comment:
do we need to answer that before merging? if not, LGTM

Reply:
Nah, that's been in there for a while and is applicable across all the signal types. 👍🏻